skimage/measure/tests/test_moments.py

```diff
@@ -10,6 +10,7 @@
 from skimage._shared import testing
 from skimage._shared.testing import (assert_equal, assert_almost_equal,
                                      assert_allclose)
+from skimage._shared.utils import _supported_float_type
```

Comment:
Since we use `_supported_float_type` in the tests below to provide the expected output type, may be it should be tested also. What do you think?

Reply:
Sure, I have added tests now